src/core/thread/mle_router.cpp

```diff
@@ -322,15 +320,11 @@ ThreadError MleRouter::HandleChildStart(otMleAttachFilter aFilter)
 
     if (mDeviceMode & ModeTlv::kModeFFD)
     {
-        advertiseDelay = Timer::SecToMsec(kReedAdvertiseInterval +
-                                          (otPlatRandomGet() % kReedAdvertiseJitter));
-        mAdvertiseTimer.Start(advertiseDelay);
+        mAdvertiseTimer.Start(
```

Comment:
That would be the same thing as a TrickleTimer that went 'inconsistent' after every firing. I would like to avoid having separate timers for each. What do you think?